### PR TITLE
feat: Gameplay event card

### PR DESCRIPTION
**Pull request by kelvinwilson-cmd**
- Local setup
- Added gameplay event card

### CODE REVIEW

File: frontend/pubspec.yaml [hunk 1]
```diff
@@ -92,6 +92,7 @@ flutter:
     - assets/images/events/
     - assets/files/
     - .env
+    - assets/images/events/gameplay.jpg
```
Comment (lukemadera) [inline frontend/pubspec.yaml:95]: You should be able to remove this, since `assets/images/events` is already set above.

Comment (kelvinwilson-cmd) [reply frontend/pubspec.yaml:95]: Thank you for pointing out the redundant line. 
I appreciate your attention to detail, I will definitely address it in my next set of code changes. : )

**Comment by kelvinwilson-cmd** [comment]
> Thanks Kelvin! Looks good overall, I noted one change. Also, can you create a branch from this repo so I can pull it down and test / run locally? And in general let's make PRs from a branch in the same repo so others can pull and test the branch easily. Finally, mark me as a reviewer in the future so I know when a new PR is ready for review!


Acknowledged!

I understand the importance of facilitating easier testing and collaboration. I will ensure to keep these in mind when taking Git Flow actions on my end.

I also got the invite- thank you; this will help streamline our workflow. 

Your guidance has been beneficial, Luke.. I appreciate it!

**Comment by lukemadera** [comment]
Glad to hear Kelvin! Sorry I didn't add you sooner - next time just let me know so I can do it. Let's close this PR and you can open a new one on a branch off this repo.

**Pull request closed without merging**